auto_inertial_params/proposal.md

```diff
@@ -0,0 +1,127 @@
+<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>
+
+# Automatically Compute Inertia Tensor Proposal
+
+* **Authors**:
+Jasmeet Singh `<jasmeet0915@gmail.com>`, Addisu Taddese `<addisu@openrobotics.org>`, Dharini Dutia `<dharinidutia@intrinsic.ai>`
+* **Status**: Draft
+* **SDFormat Version**: 1.11
+* **`libSDFormat` Version**: 14.X 
+
+## Introduction
+
+This proposal suggests adding new attributes and elements to support the automatic calculation of Moments of Inertia and Products of Inertia for a link in SDFormat 1.11. It also proposes adding support for parsing these elements and attributes in libsdformat14.
```

Comment:
Could add links to Moments of Inertia wiki

Reply:
Added a link to [this](https://en.wikipedia.org/wiki/List_of_moments_of_inertia) wiki page in 0a09b7da1822bb893b2b4aafce5d3a8c27874ff2